pkg/sql/plan2/function/operators.go

```diff
@@ -218,6 +218,18 @@ var operators = map[int][]Function{
 			TypeCheckFn: strictTypeCheck,
 			Fn:          nil,
 		},
+		{
+			Index:  15,
+			Flag:   plan.Function_STRICT,
+			Layout: COMPARISON_OPERATOR,
+			Args: []types.T{
```

Comment:
left part of =?  I don't see why we need this comment, unless here you really want to say + instead of =, but what does that really mean?  Cannot be ...

Reply:
oh, it is a incorrect notes, I will remove it